### PR TITLE
VR-5554 Support autoscaling parameter in Endpoint.update()

### PR DESCRIPTION
**Pull request by nhatsmrt**
Requires Autoscaling class implementation from https://github.com/VertaAI/modeldb/pull/1173 and metric subclasses from https://github.com/VertaAI/modeldb/pull/1177

### CODE REVIEW

**Comment by nhatsmrt** [comment]
Test passed. Uncomment.

File: client/verta/verta/_deployment/endpoint.py [hunk 1]
```diff
@@ -135,6 +136,9 @@ def update(self, run, strategy, wait=False, resources=None, autoscaling=None, en
         if not isinstance(strategy, _UpdateStrategy):
             raise TypeError("strategy must be an object from verta.deployment.strategies")
 
+        if autoscaling and not isinstance(autoscaling, Autoscaling):
+            raise TypeError("autoscaling must be an Autoscaling object")
```
Comment (convoliution) [inline client/verta/verta/_deployment/endpoint.py]:
```suggestion
            raise TypeError("`autoscaling` must be an Autoscaling object")
```

**Review by convoliution — APPROVED** [review]
Super small request

File: client/verta/verta/_deployment/endpoint.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/verta/_deployment/endpoint.py]: Fixed. Also fixed the ones for run and strategy (which I also missed :P)

**Pull request merged**